editor/gui/scene_tree_editor.cpp

```diff
@@ -361,33 +361,11 @@ void SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent) {
 	}
 
 	{
-		// Display the node name in all tooltips so that long node names can be previewed
-		// without having to rename them.
-		String tooltip = String(p_node->get_name());
-
-		if (p_node == get_scene_node() && p_node->get_scene_inherited_state().is_valid()) {
-			item->add_button(0, get_editor_theme_icon(SNAME("InstanceOptions")), BUTTON_SUBSCENE, false, TTR("Open in Editor"));
-			tooltip += String("\n" + TTR("Inherits:") + " " + p_node->get_scene_inherited_state()->get_path());
-		} else if (p_node != get_scene_node() && !p_node->get_scene_file_path().is_empty() && can_open_instance) {
-			item->add_button(0, get_editor_theme_icon(SNAME("InstanceOptions")), BUTTON_SUBSCENE, false, TTR("Open in Editor"));
-			tooltip += String("\n" + TTR("Instance:") + " " + p_node->get_scene_file_path());
+		_update_node_tooltip(p_node, item);
+		if (p_node->is_connected("editor_description_changed", callable_mp(this, &SceneTreeEditor::_update_node_tooltip))) {
+			p_node->disconnect("editor_description_changed", callable_mp(this, &SceneTreeEditor::_update_node_tooltip).bind(item));
```

Comment:
```suggestion
		const Callable update_tooltip = callable_mp(this, &SceneTreeEditor::_update_node_tooltip);
		if (p_node->is_connected("editor_description_changed", update_tooltip )) {
			p_node->disconnect("editor_description_changed", update_tooltip);
```
No need to use `bind()` for disconnecting, as it's ignored.

Note that you can reuse `update_tooltip ` Callable below by using `update_tooltip.bind()`.

Reply:
Thanks！